palm/plugins/dbt/dbt_containerizer.py

```diff
@@ -146,3 +95,33 @@ def validate_python_version(self) -> bool:
             True
         """
         return True
+
+    @classmethod
+    def determine_profile_strategy(cls, project_path:"Path") -> Tuple[str,str]:
```

Comment:
Can you explain the quotes around the `Path` typehint here? I'm not familiar with that syntax.

Also, nit on spacing here. I'm not going to comment on PEP8 stuff, just run black with `black --skip-string-normalization .`

I plan to move `palm lint` into the palm/core plugin to make this easier to enforce, and add a CI step to check formatting.

Reply:
K gonna fix the linting now. 
passing the type as a string allows you to avoid the import. I do it out of habit because I've been bit by circular import issues before; it's not totally needed here but doesn't hurt it.

Reply:
That's neat, I didn't know about that trick... I wish I did because I totally got bitten by that recently!